crates/rolldown/tests/esbuild/loader/loader_data_url_common_js_and_es6/artifacts.snap

```diff
@@ -0,0 +1,31 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/loader/loader_data_url_common_js_and_es6
+---
+# Assets
+
+## entry_js.mjs
+
+```js
+
+
+//#region y.txt
+var y_default = "data:text/plain;charset=utf-8,y";
```

Comment:
Actually this is not same with esbuild: 
```js
TestLoaderDataURLCommonJSAndES6
---------- /out.js ----------
// x.txt
var require_x = __commonJS({
  "x.txt"(exports, module) {
    module.exports = "data:text/plain;charset=utf-8,x";
  }
});

// y.txt
var y_default = "data:text/plain;charset=utf-8,y";

// entry.js
var x_url = require_x();
console.log(x_url, y_default);
```

you could open an issue to track, it is not related to dataurl encoding.